src/de/error.rs

```diff
@@ -13,7 +14,7 @@ pub enum Error {
 
     /// A [`std::string::FromUtf8Error`](https://doc.rust-lang.org/std/string/struct.FromUtf8Error.html) encountered
     /// while decoding a UTF-8 String from the input data.
-    InvalidUtf8String(string::FromUtf8Error),
+    InvalidUtf8String(Utf8Error),
```

Comment:
Changing the type contained in this error variant is a breaking change and would require a major version release, which we are not planning to do in the near term. Can a `simdutf8::simple::Utf8Error` be mapped to a `string::FromUtf8Error` to avoid this?

Reply:
The std's `String::FromUtf8Error` does not expose any constructor. So may be this can not be avoided ...

Reply:
Got it. Unfortunately we'll need to hold off on this until we can change our error types. I've added the 3.0 label to RUST-1798 to pick up this work again when we do our next major version release.